website/docs/r/pi_instance.html.markdown

```diff
@@ -127,7 +127,8 @@ Review the argument references that you can specify for your resource.
   
   Nested scheme for `pi_virtual_serial_number`:
   - `description` - (String, Optional) Description of virtual serial number.
-  - `serial` - (String, Required) Provide an existing reserved Virtual Serial Number or specify 'auto-assign' for auto generated Virtual Serial Number.
+  - `serial` - (String, Required) Provide an existing reserved Virtual Serial Number or specify 'auto-assign' for auto generated Virtual Serial Number. Updates to this will shutdown then restart power VM instance.
+  - `software_tier` - (Optional, String) Software tier for virtual serial number. Allowed values are: ["P05", "P10", "P20", "P30"]. Updates to this require VM shutdown. Updates to this will shutdown then restart power VM instance.
```

Comment:
```suggestion
  - `software_tier` - (Optional, String) Software tier for virtual serial number. Allowed values are: ["P05", "P10", "P20", "P30"]. Updates to this will shutdown then restart power VM instance.
```